lighthouse-core/gather/gatherers/source-maps.js

```diff
@@ -144,6 +161,20 @@ class SourceMaps extends Gatherer {
     }
   }
 
+  /**
+   * @param {LH.Artifacts.SourceMap} SourceMap
+   */
+  trimCommonPrefix(SourceMap) {
```

Comment:
i was expecting to see the trimming done in the audit as it's a presentational concern.

could we move it there?
i'd feel better if our sourcemap artifacts remain as we received them.

Reply:
let's do it in jsbundles. I think every audit will want this treatment.

Reply:
nvm lets do it ur way

Reply:
![image](https://user-images.githubusercontent.com/4071474/75209176-7a4bf100-5732-11ea-8bbb-3efbeaa9dce8.png)
cool